### DISCUSSION TITLE
Possible to have Nim syntax highlighting?

### DISCUSSION THREAD

**Discussion by EyeCon** [TriliumNext/Trilium#2589] [Ideas] [answered]
[Nim](https://nim-lang.org/) is a relatively new, statically typed, compiled language suitable for general purpose programming, including embedded use cases. Is it possible to have its syntax highlighting implemented for code blocks?

**Comment by zadam** [comment, marked as answer]
Hi, it looks like CodeMirror does not support Nim, so the answer is unfortunately no.

**Comment by Bullish-Design** [comment]
If I wanted to add this to a personal fork, where in the codebase/build process should I be looking?
